source/install/install-ubuntu-1804-postgresql.rst

```diff
@@ -100,9 +101,10 @@ Assume that the IP address of this server is 10.10.10.1.
 
     ``psql --host={postgres-server-IP} --dbname=mattermost --username=mmuser --password``
 
-    .. note::
-      You might have to install the PostgreSQL client software to use the command.
+.. note::
```

Comment:
@justinegeffen The last "note" section (same for the second file) is not properly formatted (minor issue).